### PR TITLE
Add Selected message to status bar

### PR DESCRIPTION
**Pull request by triplus**
As discussed:

http://forum.freecadweb.org/viewtopic.php?f=3&t=17179

### CODE REVIEW

File: src/Gui/Selection.cpp [hunk 1]
```diff
@@ -715,10 +717,18 @@ bool SelectionSingleton::addSelection(const char* pDocName, const char* pObjectN
...
         signalSelectionChanged(Chng);
 
+        snprintf(buf,512,"Selected: %s.%s.%s (%f,%f,%f)",pDocName
+                                                        ,pObjectName
+                                                        ,pSubName
+                                                        ,x,y,z);
+
 #ifdef FC_DEBUG
-        Base::Console().Log("Sel : Add Selection \"%s.%s.%s(%f,%f,%f)\"\n",pDocName,pObjectName,pSubName,x,y,z);
+        Base::Console().Log(buf);
```
Comment (DeepSOIC) [inline src/Gui/Selection.cpp]: missing \n

Comment (triplus) [inline src/Gui/Selection.cpp]: Yeah and further thinking about it i am not sure if unified approach (status bar/log) is the best approach from consistency point of view.  Therefore i will revert the original log message and that is that from my side for now.

**Comment by wwmayer** [comment]
Adding a status message with no timeout for selection actions doesn't help much in many cases because as soon as you move the mouse it can happen that the pre-selection message appears which then is again cleared after 3s.

So, the better way IMO is to remove the timeout for the pre-selection messages. See 1df3151

**Comment by triplus** [comment]
I am fine with both approaches but if you read the discussion in that thread the solution you used was the easy solution but it doesn't solve two main issues.

First: Preselected message isn't cleared (after 3s) when there is nothing under the mouse. Therefore this could count as a bug.
Second: In my opinion once user selects something that isn't pre-selection anymore.

Anyway i did my part the rest is not up to me anymore.

**Pull request closed without merging**